src/frontend/components/Felleskomponenter/InnholdContainer/InnholdContainer.tsx

```diff
@@ -13,10 +13,12 @@ const Container = styled.div`
     align-items: flex-start;
     margin: 2rem auto 20rem auto;
     color: ${navFarger.navMorkGra};
+    font-family: 'Source Sans Pro', Arial, sans-serif;
 
     && label,
     a,
-    p {
+    p,
+    li {
```

Comment:
Måtte legge til denne og font-family for at stylingen på tekst også skulle treffe bulletpoints